apps/emqx_resource/src/emqx_resource_buffer_worker.erl

```diff
@@ -1777,7 +1777,7 @@ append_queue(Id, Index, Q, Queries) ->
                 ok = replayq:ack(Q1, QAckRef),
                 Dropped = length(Items2),
                 Counters = #{dropped_queue_full => Dropped},
-                ?SLOG(info, #{
+                ?SLOG_THROTTLE(warning, #{
                     msg => "buffer_worker_overflow",
```

Comment:
the throttle ID must be an atom and enumerated in the schema

e.g. `apps/emqx_conf/src/emqx_conf_schema.erl:84:5:    cannot_publish_to_topic_due_to_not_authorized,`

Reply:
✅ A bit counterintuitive 😅